File: sets/pset6/cash/cash.py
from cs50 import get_float

while 1:
    change = get_float("Change: ")
    if change >= 0:
        break

cents = round(change * 100)
coin = [25, 10, 5, 1]
coins = 0

# Divide coins to get total number of whole coins.
for i in coin:
    coins += cents // i
    cents %= i
print(f"Coins used: {int(coins)}")

File: sets/pset1/cash/cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>

int main(){
    float change;
    int cents;
    int coins = 0;
    int type[] = {25, 10, 5, 1};
    int size = sizeof(type) / sizeof(int);

    //get positive change and convert it to whole numbers
    do{
        change = get_float("change: ");
        cents = round(change * 100);
    } while (change < 0);

   
   for (int i = 0; i < size; i++)
   {
       coins += cents / type[i];
       cents %= type[i];
   }
   
    printf ("Coins used: %i\n", coins);
}